server.go

```diff
@@ -47,30 +47,30 @@ func StartServer(port string, ipfs coreiface.CoreAPI) {
 		fmt.Println("Healthcheck server shutting down...")
 		close(idleConnsClosed)
 		if err := server.Shutdown(context.Background()); err != nil {
-			log.Printf("Healthcheck server error on Shutdown: %v", err)
+			log.Printf("Healthcheck server error on Shutdown: %+v", err)
```

Comment:
What does the plus do?

Reply:
adds field names https://pkg.go.dev/fmt#hdr-Printing
I can remove, I usually keep it there for more info, `err.String()` could work too - let me know